tests/Tests/XPack/Security/ApiKey/SecurityApiKeyTests.cs

```diff
@@ -227,5 +277,21 @@ public SecurityApiKeyTests(XPackCluster cluster, EndpointUsage usage) : base(new
 			r.PreviouslyInvalidatedApiKeys.Should().BeEmpty();
 			r.InvalidatedApiKeys.Should().HaveCount(2);
 		});
+
+		[I] public async Task SecurityClearApiKeyCacheResponse() => await Assert<ClearApiKeyCacheResponse>(ClearApiKeyCacheStep, r =>
```

Comment:
`SkipVersion` attribute?

Reply:
`Assert` skips these and returns, if the step has already been identified for skipping based on it's version range. As discussed offline, this is probably preferable.

```csharp
protected async Task Assert<TResponse>(string name, Action<TResponse> assert)
	where TResponse : class, IResponse
{
	if (_coordinatedUsage.Skips(name)) return;
	...
}
```